README.md

```diff
@@ -8,63 +8,73 @@
 
 ## Table of Contents
 
-- [Requirements](#requirements)
-- [Installation](#installation)
-  - [Platform Packages](#platform-packages)
+- [Installing](#installing)
+  - [Requirements](#requirements)
+  - [From sources](#from-sources)
   - [Godep](#godep)
+  - [Platform Packages](#platform-packages)
   - [Cross Compilation](#cross-compilation)
-- [Configuration](#configuration)
-- [Usage](#usage)
+  - [API keys](#api-keys)
+  - [Scripts](#scripts)
+- [Using](#using)
+  - [Hyphens](#hypens)
```

Comment:
Am not too sure if Hyphens might be easily picked up.
What do you think of:
```
[Hyphens: - vs --](#-vs--)
```
[Hyphens: - vs --](#-vs--)